src/corona_hakab_model/analyzers/matrix_analysis.py

```diff
@@ -13,10 +13,11 @@
 EXPORT_HISTOGRAM_FILE = "histogram_connection_type_{}.csv"
 EXPORT_HISTOGRAM_IMG = "histogram_connection_type_{}.png"
 EXPORT_MATRIX_FILE = "raw_matrix_connection_type_{}.csv"
+
 # create output directories
-EXPORT_HISTOGRAM_DIR = Path(project_structure.OUTPUT_FOLDER / "/matrix_analysis/histogram_analysis")
+EXPORT_HISTOGRAM_DIR = Path(project_structure.OUTPUT_FOLDER / "matrix_analysis/histogram_analysis")
```

Comment:
I think this could be better
```suggestion
EXPORT_HISTOGRAM_DIR = Path(project_structure.OUTPUT_FOLDER / "matrix_analysis" / "histogram_analysis")
```